db/migrations/postgres/000063_create_identities_table.up.sql

```diff
@@ -0,0 +1,143 @@
+BEGIN;
+
+CREATE TABLE identities (
+  seq                   SERIAL          PRIMARY KEY,
+  id                    UUID            NOT NULL,
+  did                   VARCHAR(256)    NOT NULL,
+  parent                UUID,
+  messages_claim        UUID            NOT NULL,
+  messages_verification UUID,
+  messages_update       UUID,
+  itype                 VARCHAR(64)     NOT NULL,
+  namespace             VARCHAR(64)     NOT NULL,
+  name                  VARCHAR(64)     NOT NULL,
+  description           VARCHAR(4096)   NOT NULL,
+  profile               TEXT,
+  created               BIGINT          NOT NULL,
+  updated               BIGINT          NOT NULL
+);
+
+CREATE UNIQUE INDEX identities_id ON identities(id);
+CREATE UNIQUE INDEX identities_did ON identities(did);
+CREATE UNIQUE INDEX identities_name ON identities(itype, namespace, name);
+
+CREATE TABLE verifiers (
+  seq            SERIAL          PRIMARY KEY,
+  hash           CHAR(64)        NOT NULL,
+  identity       UUID            NOT NULL,
+  vtype          VARCHAR(256)    NOT NULL,
+  namespace      VARCHAR(64)     NOT NULL,
+  value          TEXT            NOT NULL,
+  created        BIGINT          NOT NULL
+);
+
+CREATE UNIQUE INDEX verifiers_hash ON verifiers(hash);
+CREATE UNIQUE INDEX verifiers_value ON verifiers(vtype, namespace, value);
+CREATE UNIQUE INDEX verifiers_identity ON verifiers(identity);
```

Comment:
So we require that each verifier must have a 1:1 relationship with an identity (and an identity cannot have more than one verifier). But it is possible for an identity to exist _without_ a verifier in certain cases.

Reply:
> So we require that each verifier must have a 1:1 relationship with an identity

Not quite. `Verifier - many - one - Identity`

Reply:
Ha, well now I see why you made that comment though! Let me work out what's going on with that Index.

Reply:
The `UNIQUE` keyword is a typo - thanks for catching